qdrant_client/qdrant_client.py

```diff
@@ -49,10 +57,30 @@ def __init__(self,
 
         self._grpc_channel = None
         self._grpc_points_client = None
+        self._grpc_collections_client = None
         if prefer_grpc:
             from grpclib.client import Channel
             self._grpc_channel = Channel(host=self._host, port=self._grpc_port)
             self._grpc_points_client = grpc.PointsStub(self._grpc_channel)
+            self._grpc_collections_client = grpc.CollectionsStub(self._grpc_channel)
+
+    @property
+    def grpc_collections(self):
+        """gRPC client for collections methods
+
+        Returns:
+            An instance of raw gRPC client, generated from Protobuf
+        """
+        return self._grpc_collections_client
```

Comment:
This may be `None` when gRPC not initialized. What about checking `self._grpc_collections_client` and raising an exception with a helpful message instead of simply returning it if it is `None`.